lib/hexo/post.ts

```diff
@@ -1,12 +1,12 @@
 import assert from 'assert';
-import moment from 'moment';
 import Promise from 'bluebird';
-import { join, extname, basename } from 'path';
-import { magenta } from 'picocolors';
-import { load } from 'js-yaml';
-import { slugize, escapeRegExp } from 'hexo-util';
-import { copyDir, exists, listDir, mkdirs, readFile, rmdir, unlink, writeFile } from 'hexo-fs';
 import { parse as yfmParse, split as yfmSplit, stringify as yfmStringify } from 'hexo-front-matter';
+import { copyDir, exists, listDir, mkdirs, readFile, rmdir, unlink, writeFile } from 'hexo-fs';
+import { escapeRegExp, slugize } from 'hexo-util';
+import { load } from 'js-yaml';
+import moment from 'moment';
+import { basename, extname, join } from 'path';
+import { magenta } from 'picocolors';
 const preservedKeys = ['title', 'slug', 'path', 'layout', 'date', 'content'];
```

Comment:
Would you mind restoring the import order?

Reply:
sorry that auto optimized imports by vscode. ill restore it.

Reply:
restored at https://github.com/dimaslanjaka/hexo/commit/6d5bb6442d639dc96428e916f3f12dd5bc80cb64